starknet_py/tests/e2e/fixtures/contracts_v1.py

```diff
@@ -268,3 +266,131 @@ async def simple_storage_with_event_contract(
+
+    return resp.class_hash
+
+
+async def declare_cairo1_account(
+    account: BaseAccount,
+    compiled_account_contract: str,
+    compiled_account_contract_casm: str,
+) -> int:
+    """
+    Declares a specified Cairo1 account.
+    """
+
+    casm_class = create_casm_class(compiled_account_contract_casm)
+    casm_class_hash = compute_casm_class_hash(casm_class)
+    declare_v2_transaction = await account.sign_declare_v2(
+        compiled_contract=compiled_account_contract,
+        compiled_class_hash=casm_class_hash,
+        max_fee=MAX_FEE,
+    )
+    resp = await account.client.declare(transaction=declare_v2_transaction)
+    await account.client.wait_for_tx(resp.transaction_hash)
+    return resp.class_hash
+
+
+@pytest_asyncio.fixture(scope="package")
+async def account_with_validate_deploy_class_hash(
+    pre_deployed_account_with_validate_deploy: BaseAccount,
+) -> int:
+    contract = load_contract("Account")
+    casm_class_hash = compute_casm_class_hash(create_casm_class(contract["casm"]))
+
+    return await declare_account(
+        pre_deployed_account_with_validate_deploy, contract["sierra"], casm_class_hash
+    )
+
+
+@pytest_asyncio.fixture(scope="package")
+async def argent_cairo1_account_class_hash(
+    account: BaseAccount,
+) -> int:
+    # Use precompiled argent account contracts
+    # we don't have the source code for this contract
+    compiled_contract = read_contract(
+        "argent_account.json", directory=PRECOMPILED_CONTRACTS_DIR
+    )
+    compiled_contract_casm = read_contract(
+        "argent_account.casm", directory=PRECOMPILED_CONTRACTS_DIR
+    )
+    return await declare_cairo1_account(
+        account=account,
+        compiled_account_contract=compiled_contract,
+        compiled_account_contract_casm=compiled_contract_casm,
+    )
+
+
+async def deploy_v1_contract(
+    account: BaseAccount,
+    contract_name: str,
+    class_hash: int,
+    calldata: Optional[Dict[str, Any]] = None,
+) -> Contract:
+    """
+    Deploys Cairo1 contract.
+
+    :param account: An account which will be used to deploy the Contract.
+    :param contract_name: Name of the contract from project mocks (e.g. `ERC20`).
+    :param class_hash: class_hash of the contract to be deployed.
+    :param calldata: Dict with constructor arguments (can be empty).
+    :returns: Instance of the deployed contract.
+    """
+    contract_sierra = load_contract(contract_name)["sierra"]
+
+    abi = create_sierra_compiled_contract(compiled_contract=contract_sierra).parsed_abi
+
+    deployer = Deployer()
+    deploy_call, address = deployer.create_contract_deployment(
+        class_hash=class_hash,
+        abi=abi,
+        calldata=calldata,
+        cairo_version=1,
+    )
+    res = await account.execute_v1(calls=deploy_call, max_fee=MAX_FEE)
+    await account.client.wait_for_tx(res.transaction_hash)
+
+    return Contract(address, abi, provider=account, cairo_version=1)
```

Comment:
nit: `1` is the default value for `cairo_version` param, so we can remove extra arg here.